### ISSUE TITLE
settings.setDefaultServiceType results in undefined ServiceType value

### ISSUE THREAD

**Issue by buchatsky** [umutozel/Beetle.js#4]
Reporting: a bug
```
            var defType = settings.getDefaultServiceType(); // enums.serviceTypes.Beetle
            settings.setDefaultServiceType(enums.serviceTypes.OData);
            defType = settings.getDefaultServiceType(); // undefined

```
My workaround - modifying beetle.js:
```
        function getValue(value, type, typeEnum) {
                // ...
                if (Assert.isEnum(value, typeEnum)) return value;//.instance;
                // ...
        }

```
(or creating ODataService explicitly)

**Versions:**
beetle.js:  3.0.6

**Comment by umutozel** [comment]
Thank you for reporting. Last few weeks were really busy, I'll try to fix and release a new npm this week.

**Comment by umutozel** [comment]
Fixed with 3.0.7. Thanks again for reporting and sorry for the delay.
